lisa/wa_results_collector.py

```diff
@@ -492,6 +492,48 @@ def row_filter(r): return True
 
         return ret
 
+    def _read_energy_instrument_metrics(self, path):
+        """
+        Parse the WA energy_measurement output.
+        """
+        path_cache = path + ".parquet"
+
+        if not os.path.isfile(path_cache):
```

Comment:
rather than `os.path.isfile`, you can try to read the file and handled the exception. That's the preferred pattern as it avoids race condition and also duplication of assumptions (read_parquet is already aware of what it can handle)

Reply:
Done